api-reference/user-app/records/records.yaml

```diff
@@ -113,7 +113,7 @@ paths:
                                                 - SCHEDULED
                                                 - ERRORED
                                                 - RESCHEDULED
-                                    patientOid:
+                                    patient_id:
```

Comment:
this is the patient_oid i guess.

Reply:
we are using patient_id in most of the place so thought to make it same across all api